.circleci/config.yml

```diff
@@ -35,7 +35,7 @@ jobs:
                 command: rustup component add clippy
             - run:
                 name: Run Clippy
-                command: cargo clippy -- --deny warnings
+                command: cargo clippy --package trin -- --deny warnings
                 no_output_timeout: 20m
```

Comment:
If this is going to shorten clippies time you can probably also just remove ``no_output_timeout: 20m`` in the same PR because we shouldn't need the extra time anymore.

Also maybe the comment on why I did it, but I guess that is optional.